### PR TITLE
Upgrade package set for Try PureScript

### PR DESCRIPTION
**Pull request by thomashoneyman**
The package set for Try PureScript is from May of this year. Plenty of packages have been added or upgraded in that time, and I thought that the eve of the 0.14 release would be a good time to get a new package set up and running on Try PureScript.

I'm not sure how to verify the externs sizes against the Try PureScript machine, but I seriously doubt the new libraries and their externs push us over the limit.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -18,37 +18,49 @@
...
+  - Options are: `true`, `false` (default)
+  - Example: `js=true` will print JavaScript code instead of the program's output
 
 - **Session**: Load code from a session which is stored with [localStorage](https://developer.mozilla.org/en-US/docs/Web/API/Window/localStorage) using the `session` parameter
-    - Usually managed by Try PureScript
-    - Example: `session=9162f098-070f-4053-60ea-eba47021450d` (Note: will probably not work for you)
-    - When used with the `gist` query parameter the code will be loaded from the Gist and not the session
+  - Usually managed by Try PureScript
+  - Example: `session=9162f098-070f-4053-60ea-eba47021450d` (Note: will probably not work for you)
+  - When used with the `gist` query parameter the code will be loaded from the Gist and not the session
```
Comment (thomashoneyman) [inline README.md:63]: I'm sorry about the autoformatting. I can revert it if desired and there are no changes to this part of the documentation.

Comment (hdgarrood) [reply README.md:63]: Yes please (sorry for the bother)

**Comment by hdgarrood** [comment]
We can't make assumptions about memory; right now the graphs in DO show that 60% of the memory of the server is in use. I expect that libraries generally increase in size over time too. I'll try to remember how I checked.

File: README.md [hunk 2]
```diff
@@ -18,37 +18,49 @@
...
+```
+
+Then, install every package in the set:
 
 ```
 $ spago ls packages | cut -f 1 -d ' ' | xargs spago install
 ```
 
-to install every package in the set. Before deploying an updated package set, someone should check that the memory required to hold the package set's externs files does not exceed that of the try.purescript.org server.
+If this command fails because a package that previously existed in the package set has been removed, then edit the `spago.dhall` file to remove the offending packages (you can also just remove all dependencies). Then, re-run the installation command above.
```
[the remark this answers is not in the document]
Comment (thomashoneyman) [reply README.md:33]: Yes, this should probably just direct contributors to remove all the dependencies. Doing so does produce an error, though:

```
λ spago build                                   
spago: 
↳ ./spago.dhall

Error: An empty list requires a type annotation

2│                  []

/.../trypurescript/staging/spago.dhall:2:18
```

In Dhall the type annotation would be `List Text`, so we'd have to suggest setting the dependencies to `[] : List Text` before running the `$ spago ls packages ...` command.

[the remark this answers is not in the document]
Comment (thomashoneyman) [reply README.md]: I've made the required steps more clear in the latest commit.

**Pull request closed without merging**